### PR TITLE
Expand copter tailsitter information

### PR DESCRIPTION
**Pull request by Hwurzburg**
@kd0aij Mark, I need info on how the TRI is setup....does it allow tilt yaw? how are motors arranged, numbered, and rotations...so I can draw it like the others....please check the diagrams...

I had already added some copter tailsitter info and just forgotten it....been typing a lot lately...

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@kd0aij updated motor diagram....ready for final review

File: plane/source/docs/guide-tailsitter.rst [hunk 1]
```diff
@@ -50,7 +53,52 @@ QHOVER and QLOITER.
...
+QUAD X Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
+
+.. image:: ../images/x-copter-quadplane.jpg
+
+
+Tricopter Tailsitter
+--------------------
+Motors are controller by the M1/M2/M4 outputs:
```
Comment (kd0aij) [inline plane/source/docs/guide-tailsitter.rst]: the only testing I know of for this configuration is my SITL testing, which has motors 1 and 2 on the wings and motor 4 on the top of the tail fin

File: plane/source/docs/guide-tailsitter.rst [hunk 2]
```diff
@@ -50,7 +53,52 @@ QHOVER and QLOITER.
...
+QUAD I Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
+
+.. image:: ../images/i-copter-quadplane.jpg
+
+
+QUAD X Motor Tailsitter
+-----------------------
+Motors are controlled by the M1-M4 outputs:
```
Comment (kd0aij) [inline plane/source/docs/guide-tailsitter.rst]: the only example of this that I know of is @losawing's bi-wing which has motors 1 and 3 on the lower wing and 2,4 on the top wing. Also, he is using a PR branch which does use differential torque for yaw and hence the standard copter motor layout and rotation directions.

File: plane/source/docs/guide-tailsitter.rst [hunk 3]
```diff
@@ -50,7 +53,52 @@ QHOVER and QLOITER.
...
+
+Single/Dual Motor Tailsitter
+----------------------------
+
+Motors are controlled by the Throttle, Throttle Left, Throttle Right outputs:
+
+.. image:: ../images/tailsit-motors.jpg
+
+
+QUAD I Motor Tailsitter
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply plane/source/docs/guide-tailsitter.rst]: brain glitch...meant + not I

File: plane/source/docs/guide-tailsitter.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/guide-tailsitter.rst]: yes, need to swap 1 and 2...

File: plane/source/docs/guide-tailsitter.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/guide-tailsitter.rst]: but this is correct as far as you know for 4.0, right?

File: plane/source/docs/guide-tailsitter.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply plane/source/docs/guide-tailsitter.rst]: fixed

File: plane/source/docs/guide-tailsitter.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/guide-tailsitter.rst]: did that

**Comment by Hwurzburg** [comment]
fixed review comments...should be ready now...thanks

File: plane/source/docs/guide-tailsitter.rst (hunk 2 above)
Comment (kd0aij) [reply plane/source/docs/guide-tailsitter.rst]: yes, just untested

**Pull request merged**